test/functional/operation_generators_example.test.js

```diff
@@ -2931,6 +2931,7 @@ describe('Operation (Generators)', function() {
           .newClient(configuration.writeConcernMax(), { poolSize: 1 })
           .connect();
         var db = client.db(configuration.db);
+        yield db.createCollection('example');
```

Comment:
what's motivating this change?

Reply:
This test fails if you run it in isolation, it depends upon other tests to create collections.